[VAN-126178] Added sample oauth for python-sdk. (#33)
visier/api-samples: authentication/oauth2-password/python/sdk/README.md

# Authentication With Python Connector and Password
1. Create a virtual environment: `python3 -m venv env`.
1. Activate the virtual environment: `source env/bin/activate`.
1. Install the required dependencies: `pip install -r requirements.txt`.
1. Clone the `../.env-password-template` environment file to `.env`.
1. Update the `.env` file with the appropriate values to match your system details.

> 1. Source the environment file `source .env` and provide the sensitive values. Sensitive values should not be stored in the 'source .env` file itself.

Reviewer: Consider:
```suggestion
1. Source the environment file by executing `source .env` and provide the sensitive values when prompted. Sensitive values should not be stored in the `.env` file itself.
```
